.cirrus.yml

```diff
@@ -1,6 +1,9 @@
 lint_task:
   container:
     image: python:3.7-slim
+  env:
+    LC_ALL: C.UTF-8
```

Comment:
I would have actually fixed this in `setup.py`

Reply:
How?

Reply:
`open(<file>, encoding='utf-8')`

For more details, see https://docs.python.org/3/library/functions.html#open